src/main/scala/nl/biopet/utils/biowdl/references/Reference.scala

```diff
@@ -59,4 +59,6 @@ trait Reference {
   def starGenomeDir: Option[File] = None
   def hisat2Index: Option[String] = None
 
+  def referenceGtf: Option[File] = None
```

Comment:
This should not be part of reference, this 1 reference can have multiple annotations.
Better make a trait Annotation like this:
```scala
trait Annotation extends Reference {
  //Do things
}
```